lib/metrics/statsd.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var StatsD = require('node-statsd')
+
+var STATSD_PREFIX = 'fxa.auth.'
+
+function StatsDCollector(log) {
+  if (! log) {
+    throw new Error('Log is required')
+  }
+
+  var config = require('../../config')
```

Comment:
For stylistic consistency with the rest of the codebase, we should pass in `config` to the constructor rather than requiring it here.